### PR TITLE
bugfix/#3199 "Дані користувача" tab differs from mockups 

### PR DESCRIPTION
**Pull request by nazarf1**
'Дані користувача' tab matches the mockups

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 1]
```diff
@@ -97,6 +104,14 @@
   }
 }
 
+.form-group {
+  p {
+    padding: 13px 13px 13px 0px;
+    color: #000000;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:110]: We use variables for colors, so try to find an appropriate one and use it here

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 2]
```diff
@@ -49,12 +48,20 @@
...
   h5 {
+    font-weight: bold;
     margin-left: 15px;
+    margin-bottom: 30px;
+    margin-top: 30px;
+  }
+
+  h3 {
+    font-weight: bold;
+    color: #000000;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:64]: We use variables for colors, so try to find an appropriate one and use it here

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss [hunk 3]
```diff
@@ -49,12 +48,20 @@
...
-    margin: 30px;
+    margin: 30px 0 30px 30px;
     font-family: var(--secondary-font);
   }
 
   h5 {
+    font-weight: bold;
     margin-left: 15px;
+    margin-bottom: 30px;
+    margin-top: 30px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss:59]: Replace this with:
margin: 30px 0 30px 15px;

Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: done

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 2 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: done

File: src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss (hunk 1 above)
Comment (nazarf1) [reply src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss]: done

**Review by Bishko86 — APPROVED** [review]

**Review by nazarhlyvka — APPROVED** [review]

**Review by rostikdoroshenko — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

**Review by KhrystynaPr — APPROVED** [review]

**Review by NewArtMask — APPROVED** [review]

**Review by s1auka — APPROVED** [review]

**Review by nazarhlyvka — APPROVED** [review]

**Pull request merged**